ui/lobby/css/_setup.scss

```diff
@@ -27,7 +28,8 @@ $game-family-random-pieces: (
   'mixed.svg',
   'mixed.svg',
   'bwS.svg',
-  'wbS.svg'
+  'wbS.svg',
+  'wbK.svg' // @TODO: change
```

Comment:
What tool would you use to edit an SVG (for creating a bicolored staunty pawn) ?

Reply:
I think I've made them png in the past as it's easier without any tools, just used paint3d.